modules/pel/peltool/peltool.py

```diff
@@ -540,7 +540,7 @@ def main():
     parser.add_argument('-o', '--output-dir', dest='output_dir',
                         help='Directory to write output files when processing a directory')
     parser.add_argument('-e', '--extension', dest='extension',
-                        help='Used with -d, only look for files with this extension (e.g. ".pel")')
+                        help='Only look for files with this extension (e.g. ".pel")')
```

Comment:
I missed this...

Is it usable for other options or only for -j/--json?

If it's only for the JSON option, we should use the argparse group as we discussed.

Reply:
It's usable for other options as well. 
For example, if used along with -l, we would consider files only with passed extension and ignore others.

Reply:
> It's usable for other options as well. For example, if used along with -l, we would consider files only with passed extension and ignore others.

It seems that we are not offering any extensions for the PELs file. I suspect that the user of the Python tool is applying an extension to the PELs file they receive from BMC/Dump before inputting it into the Python peltool.

@spinler is it correct understanding?

Reply:
This was for the case where the PELs were coming back from the HMC in a 'iqqypell' or maybe 'iqqymerge' file where the HMC puts PEL binary data they got from the BMC, and then some backend code was pulling them out and giving them an extension.

Any PELs coming directly from the BMC won't have an extension though.

Reply:
Harsh, I believe we should keep this simple and only support the -j/--json option. However, if you think it will work for other options as well, I'm okay with that.

Reply:
Yeah! This will work with other option as well.